src/Pages/AuthPages/SignupPage.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import './AuthPage.css'
+import Navbar from '../../Components/Navbar/Navbar'
+import { Link, useNavigate } from 'react-router-dom'
+import { useState } from 'react'
+import FormInput from './AuthComponents/FormInput'
+const axios = require('axios');
+
+const SignupPage = () => {
+    const navigate = useNavigate()
+    const [userExistError, setUserExistError] = useState(false);
+    const [signupError, setSignupError] = useState(false);
+    const [signupData, setSignupData] = useState({
+        firstName: '', lastName: '', email: '', password: '',
+    });
+    const handleUserSignup = async (e) => {
+        e.preventDefault()
+        try {
+            const res = await axios.post(
+                '/api/auth/signup',
+                { ...signupData }
+            )
+            console.log(res)
+            if (res.status === 200 || res.status === 201) {
```

Comment:
changes: remove console.log() when you push code to github

Reply:
done